package.json

```
@@ -112,6 +113,7 @@
     "morgan": "^1.9.1",
     "nanoid": "^2.0.0",
     "path-to-regexp": "^2.4.0",
+    "polyfill-crypto.getrandomvalues": "^1.0.0",
```

```suggestion
    "polyfill-crypto.getrandomvalues": "1.0.0",
```

Considering [polyfill-crypto.getrandomvalues](https://www.npmjs.com/package/polyfill-crypto.getrandomvalues) has a single contributor and less than a 1000 weekly downloads should we limit to using v1.0.0? This would stop something potential malicious coming through in a patch version without us explicitly bumping the package. 

cc @ChrisBAshton @sareh - thoughts?

---

Seems fair 👍

---

Done :)